src/tests.js

```diff
@@ -64,11 +54,18 @@ function Test(component, config) {
     }
   }
 
+  const mixins = config.mixins === undefined ? {} : config.mixins
+
   return testComponent.mixin({
     find: Find,
     setState: SetState,
-    simulate: Simulate
+    simulate: Simulate,
+    ...mixins
   })
 }
 
+export const addMixins = (Test, mixins) => {
+  return (component, config) => Test(component, { ...config, mixins })
+}
```

Comment:
@zackify How about this ?? `CustomTest = addMixins(Test, mixins)` Then just use `CustomTest(<TestComponent />)`  I will add tests after a few hours.